core/src/androidMain/kotlin/Scanner.kt

```diff
@@ -47,7 +51,17 @@ public class AndroidScanner internal constructor() : Scanner {
                 cancel("Bluetooth scan failed", ScanFailedException(errorCode))
             }
         }
-        bluetoothAdapter.bluetoothLeScanner.startScan(callback)
+
+    val scanFilter =
+        filterServices
+            ?.map { ScanFilter.Builder().setServiceUuid(ParcelUuid(it)).build() }
+            ?.toList()
```

Comment:
Wups, when I suggested this change, I think I suggested the wrong indent.

```suggestion
        val scanFilter =
            filterServices
                ?.map { ScanFilter.Builder().setServiceUuid(ParcelUuid(it)).build() }
                ?.toList()
```